src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -505,19 +505,13 @@ protected DynamoModel(IStartConfiguration config)
 
             DisposeLogic.IsShuttingDown = false;
 
-            // Create a core which is used for parsing code and loading libraries
-            var libraryCore =
-                new ProtoCore.Core(new Options { RootCustomPropertyFilterPathName = string.Empty });
-
-            libraryCore.Compilers.Add(ProtoCore.Language.kAssociative, new ProtoAssociative.Compiler(libraryCore));
-            libraryCore.Compilers.Add(ProtoCore.Language.kImperative, new ProtoImperative.Compiler(libraryCore));
-            libraryCore.ParsingMode = ParseMode.AllowNonAssignment;
-
-            LibraryServices = new LibraryServices(libraryCore, pathManager);
+            LibraryServices = new LibraryServices(pathManager);
             LibraryServices.MessageLogged += LogMessage;
             LibraryServices.LibraryLoaded += LibraryLoaded;
 
             ResetEngineInternal();
+            LibraryServices.LoadLibraries();
```

Comment:
Its already loaded in thge EngineController constructor: libraryServices.SetLibraryCore(LiveRunnerCore);